OpenEdXMobile/src/main/java/org/edx/mobile/player/PlayerFragment.java

```diff
@@ -900,8 +900,14 @@ public void onFullScreen(boolean isFullScreen) {
             isManualFullscreen = isFullScreen;
             if (isFullScreen) {
                 enterFullScreen();
+                environment.getAnalyticsRegistry().trackVideoOrientation(videoEntry.videoId,
+                        player.getCurrentPosition() / AppConstants.MILLISECONDS_PER_SECOND,
+                        true, videoEntry.eid, videoEntry.lmsUrl);
             } else {
                 exitFullScreen();
+                environment.getAnalyticsRegistry().trackVideoOrientation(videoEntry.videoId,
+                        player.getCurrentPosition() / AppConstants.MILLISECONDS_PER_SECOND,
+                        false, videoEntry.eid, videoEntry.lmsUrl);
```

Comment:
We can replace the 3rd argument in this function call with `isFullscreen` and move this call at the bottom of the parent `if` i.e. `if (isPrepared)`. This way we can get rid of the extra function call above as well.